.gitconfig

```diff
@@ -75,6 +75,9 @@
 			git commit --amend -m \"$(git log -1 --pretty=%B)\n\nCloses #$1.\"; \
 		fi \
 	}; f"
+	
+	# List git aliases
+	aliases = config --get-regexp alias
```

Comment:
I use this which removes the alias. prefix and excludes the alias command from the returned list.

`alias = ! git config --get-regexp ^alias\\. | sed -e s/^alias\\.// -e s/\\ /\\ =\\ / | grep -v ^'alias '`

```
$ git alias
br = branch
ci = commit
```